cmd/osbuild-composer/composer.go

```diff
@@ -200,7 +199,17 @@ func (c *Composer) Start() error {
 		}()
 	}
 
-	return c.weldr.Serve(c.weldrListener)
+	if c.weldrListener != nil {
+		go func() {
+			err := c.weldr.Serve(c.weldrListener)
+			if err != nil {
+				panic(err)
+			}
+		}()
+	}
+
+	// wait indefinitely
+	select {}
```

Comment:
Note: we probably want to stop when any of the APIs `panic`, which is easily solvable by passing a channel to each of the above goroutines. Out of scope for this PR.

Reply:
I think that's the default Go behaviour: https://play.golang.org/p/pL4FVhEZ3-c